docs/src/conf.py

```diff
@@ -233,4 +233,6 @@ def find_version(*file_paths):
     "userguide/intro": "index.html",
     # Move quickstart into userguide
     "quickstart/*": "/userguide/${source}.html",
+    # Move io under pipelines
```

Comment:
The IO sidebar nav was only one page; and to me, it makes more sense to be nested under Pipelines.

Reply:
(again, no content was changed for the file that was moved)